docs/ACCESSIBILITY.md

```diff
@@ -211,6 +211,22 @@ To make the live region more consistent across browsers and easier to control, w
    -  It is okay to disable all buttons, as long as the answer will be read by the screen reader
    -  Related to [#3135](https://github.com/microsoft/BotFramework-WebChat/issues/3135)
 -  Don't move focus when an activity arrives (or asynchronously)
-   -  Screen reader reading will be interrupted when focus change
+   -  Screen reader reading will be interrupted when focus changes
    -  Only change focus synchronous to user gesture
    -  Related to [#3135](https://github.com/microsoft/BotFramework-WebChat/issues/3135)
+
+# Screen reader renderer for custom activities and attachments
+
+Web Chat render components are accompanied by a screen reader renderer to maximize accessibility. In the case of custom components, the bot/Web Chat developer will need to implement a screen reader renderer for the equivalent custom visual component.
+
+![image: Console warning: "No renderer for attachment for screen reader of type "application/mnd.microsoft.card.adaptive"](https://user-images.githubusercontent.com/14900841/106323546-6f47ed80-622c-11eb-96d7-de6f72818525.png)
+
+The Web Chat team **DOES NOT** recommend disabling warning messages regarding screen readers and accessibility. However, if the developer decides to suppress these messages, it can be done by adding the following code to `attachmentForScreenReaderMiddleware` in the `Composer` props.
+
+```js
+const attachmentForScreenReaderMiddleware = () => next => (...args) => {
+   return false;
```

Comment:
do we have a test covering this case attachmentForScreenReaderMiddleware return false  and  validation we are not getting warning messages

Reply:
yep, the test already exists at `__tests__/html/middleware.liveRegionAttachment.noWarning.returnFalse.html`